docs/source/_developers/primer.md

```diff
@@ -0,0 +1,234 @@
+- Keep your writing easy to read by using short sentences and paragraphs.
+- If you need to explain complex concepts, break them into smaller, more manageable sections to help your reader understand your message.
+
+##  Formatting
+MyST allows for markdown to be used for our Spinx project (i.e., RTD).
+
+:::{admonition} Bookmark and Review-worth Reference
+Check out this [Markdown cheatsheet].
+:::
+
+
+Commonly used elements follow the [MyST-Typography].
+
+Directives allow us to insert and format various elements (e.g., images, callouts, etc.). We use `colon_fence` to replace the typical hyphens Learn more about  [Colon Fence].
+
+:::::{important}
+We use the `colon_fence` extension the markdown rendered by Myst parses three colons (i.e., *:*) to open and close blocks. ([about colon_fence]).
+
+::::{note}
+[Myst-Parser Documentation] includes several constructs we can use to enhance our pages. **Use elements with purpose, whether to call attention, reference related content, or warn.
+::::
+
+Nesting blocks, along with the ability set parameters inline.
+
+::::{note}
+There is no need to indent source text, as nested directives are explicitly clear by the number of `:` characters are used to open and close.
+::::
+
+::::{attention}
+
+**Do not add line-breaks unless a line break in the rendered view is to need a line break.** Each paragraph remains on the same line, with line-breaks using to clearly show transitions between paragraphs, sections, and directives.
+
+:::{tip}
+Set *Soft-Wrap* to be set to 100 characters by default when viewing `*.md`
+
+<figure>
+  <img src="../_static/image/settings-softwrap.png" alt="Soft-Wrap Setting" style="width:100%">
+  <figcaption>Fig. Soft-Wrap set to 100 characters in PyCharm's <i>General</i> settings.</figcaption>
+</figure>
+:::
+
+:::{note}
+Add line-breaks before and after the opening and closing of directives (i.e., outside the struct). Only add a single line-break after opening, which is the case for most directives; for those that do not require any inputs but titles (e.g., `:::{directive} Title`), omit line-breaks after the opening.
+:::
+::::
+
+::::{important}
+These nested admonitions were done, mostly, to demonstrate how it can be done. There are exceptions, but this is quite nested: be sure to consider if the flow of information is done most effectively when nests deeper, or even as as deep as this example (i.e., five colons `:::::` deep).
+::::
+:::::
+
+Consistent formatting makes your documentation look professional and easy to read. Use consistent formatting for headings, lists, tables, and other elements, and adequately nest subheadings and lists. Here are some guidelines:
+
+- **Headings**: Use headings and subheadings to structure your content. In MyST, use # for level 1 headings, ## for level 2, and so on.
+- **Lists:** Use bulleted lists for unordered items and numbered lists for ordered items. In MyST, use or  for unordered lists and 1. for ordered lists.
```

Comment:
When should we use a  numbered list and when not to? The conda page originally had instructions as numbered lists, I kind of prefer it not to, but the R packrat page does have numbered lists as do other pages.....so a consensus here would be good. How do we want a set of instructions that are meant to be followed in order look?

Reply:
Ordered lists are numbered; unordered are bullets (I fixed that sentence, it seemed to have lost text)... see if that is better